core/lib/libtomcrypt/sub.mk

```diff
@@ -114,6 +114,7 @@ _CFG_CRYPTO_WITH_ASN1 := $(call cryp-one-enabled, RSA DSA ECC)
 _CFG_CRYPTO_WITH_FORTUNA_PRNG := $(call cryp-all-enabled, AES SHA256)
 
 cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -DLTC_SMALL_CODE -DLTC_NO_FAST
+cppflags-lib-$(CFG_CRYPTO_SIZE_OPTIMIZATION) += -Os
```

Comment:
Indeed, `LTC_NO_FAST` resets `LTC_FAST` if the latter has been set. Indeed, `LTC_FAST` is never set on ARM architecture, so `LTC_NO_FAST` has no effect.

Anyway, setting `LTC_NO_FAST` does not hurt.